com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -1,25 +1,27 @@
 # Changes in Alembic for Unity
 ## [2.2.0-exp.3] - 2021-01-22
-### Changes
+### Changed
 - Minimum Unity version is 2019.4
-
 ### Fixed
 - Fixed a bug that caused the Alembic exporter to fail if GameObjects were being deleted during the recording session.
 - Fixed a memory leak that occurred when using varying topology meshes.
 
 ## [2.2.0-exp.2] - 2021-01-21
-### Changes
+### Changed
 - Updated to Alembic version 1.7.16.
+- The package depends on the Cloth Unity Module.
+- Renamed AlembicCurve CurvePointCount to CurveOffsets, and changed the semantic to a stride array.
+
+### Added
 - Added support for Stadia standalone builds.
 - Added support for arm64 macOS.
```

Comment:
Just invert the order of these sections. We should ideally have:
- Added
- Changed
- Fixed